### DISCUSSION TITLE
Setting Up Wood as a solid fuel

### DISCUSSION THREAD

**Discussion by MARNACARAHMAN** [firemodels/fds#10413] [General]
hello. i need help for setting up wood as solid fuel source. i need this for my thesis. can anyone give or tell me the codes for settinf up wood as fire source. please

**Comment by rmcdermo** [comment]
Please read the user guide and post any further questions to the [FDS discussion forum](https://groups.google.com/g/fds-smv).

This GitHub Discussions feature is not yet an official channel of communication and so the Google discussion group is going to give you more access to help right now.
